src/components/Modal/Modal.jsx

```diff
@@ -121,7 +122,7 @@ Modal.propTypes = {
    */
   children: PropTypes.node,
   /**
-   * Reference to close button element. It is used to close modal when Escape key is pressed or the backdrop is clicked.
+   * Reference to close button element. It is used to close modal when Escape key is pressed.
```

Comment:
⚠️  This means we are losing functionality of closing the Modal by clicking on backdrop. That has a big impact on UX.

IIRC, we added this functionality to Spirit Modal later, so maybe it's not that easy.

I'd like to ask you to:

- either bring back/re-implement the functionality of closing the dialog on backdrop click (watch out for "nested" dialogs),
- or file an issue that we get on rather sooner than later.

https://github.com/lmc-eu/spirit-design-system/blob/main/packages/web-react/src/hooks/useClickOutside.ts

https://github.com/lmc-eu/spirit-design-system/tree/main/packages/web-react/src/components/Modal#stacking-modals

https://spirit-design-system.netlify.app/packages/web-react/src/components/modal/

Reply:
On call we agreed that for now we do not need this functionality.

Not having it has the added benefit of not loosing unsaved work in modal on accidental click.
If we need it in the future it will be added via a new issue.

Reply:
I introduced it back with possibility to disable it as decided on Monday's call.